kaspad/src/emulator.rs

```diff
@@ -51,14 +51,11 @@ impl RandomBlockEmitter {
 
         let mut tips = vec![self.genesis];
         let mut total = 0;
-        let mut timestamp = 0u64;
 
         while total < self.target_blocks {
             let v = min(self.max_block_parents, poi.sample(&mut thread_rng) as u64);
-            timestamp += (self.delay as u64) * 1000;
-            if v == 0 {
```

Comment:
We need to keep this since otherwise `new_tips` below becomes empty and the next round has no parents

Reply:
yes, i thought i already pushed this back in, i removed it cause i thought it was some timestamp check due to placement of the code.

Reply:
okay it is back in.